cassandra/setup.py

```diff
@@ -27,6 +27,9 @@ def get_dependencies():
         return f.readlines()
 
 
+CHECKS_BASE_REQ = 'datadog-checks-base>=11.0.0'
```

Comment:
JMX integration like `activemq`

```suggestion
CHECKS_BASE_REQ = 'datadog-checks-base'
```